<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="91" started-at="2018-12-01T11:49:41Z" finished-at="2018-12-01T11:49:41Z">
    <groups>
    </groups>
    <test name="Test1" duration-ms="91" started-at="2018-12-01T11:49:41Z" finished-at="2018-12-01T11:49:41Z">
      <class name="com.parallel.tests.HomePageTest1">
        <test-method status="FAIL" signature="Test(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.parallel.tests.HomePageTest1@cf57e3]" name="Test" duration-ms="0" started-at="2018-12-01T17:19:41Z" data-provider="getData" finished-at="2018-12-01T17:19:41Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Data Provider public java.lang.Object[][] com.parallel.tests.HomePageTest1.getData() must return either Object[][] or Iterator<Object>[], not class [[Ljava.lang.Object;]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[][] com.parallel.tests.HomePageTest1.getData() must return either Object[][] or Iterator<Object>[], not class [[Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:509)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test -->
      </class> <!-- com.parallel.tests.HomePageTest1 -->
    </test> <!-- Test1 -->
    <test name="Test2" duration-ms="91" started-at="2018-12-01T11:49:41Z" finished-at="2018-12-01T11:49:41Z">
      <class name="com.parallel.tests.HomePageTest">
        <test-method status="FAIL" signature="Test(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.parallel.tests.HomePageTest@12fad11]" name="Test" duration-ms="0" started-at="2018-12-01T17:19:41Z" data-provider="getData" finished-at="2018-12-01T17:19:41Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Data Provider public java.lang.Object[][] com.parallel.tests.HomePageTest.getData() must return either Object[][] or Iterator<Object>[], not class [[Ljava.lang.Object;]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[][] com.parallel.tests.HomePageTest.getData() must return either Object[][] or Iterator<Object>[], not class [[Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:509)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test -->
      </class> <!-- com.parallel.tests.HomePageTest -->
    </test> <!-- Test2 -->
  </suite> <!-- Suite -->
</testng-results>
